clients/audacious/audplugin.cc

```diff
@@ -89,14 +89,16 @@ string imms_get_playlist_item(int at)
 
 static void player_reset_selection()
 {
-    if (next_plpos != -1) {
-        auto pl = Playlist::playing_playlist();
-        if (next_plpos < pl.n_entries()) {
-            int qp = pl.queue_find_entry(next_plpos);
-            if (qp >= 0)
-                pl.queue_remove(qp);
-            next_plpos = -1;
-        }
+    if (next_plpos < 0) {
```

Comment:
This change should be squashed together (using "fixup") with the earlier change touching the same code.